src/amo/components/Search/styles.scss

```diff
@@ -0,0 +1,56 @@
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+
+.Search {
+  @include respond-to(large) {
+    display: grid;
+    grid-auto-flow: column dense;
+    grid-gap: 10px;
+    grid-template-columns: minmax(300px, 35%) auto;
+    padding: $padding-page;
+  }
+
+  .Card {
+    margin: $padding-page;
+
+    @include respond-to(large) {
+      margin: 0;
+    }
+  }
+
+  .SearchSort {
```

Comment:
They do in order to be scoped to this particular container; I didn't want the `SearchSort` to have these styles by default–I figured with things like margins we can set them in the container's styles scoped like this so different containers could position it slightly differently.

That said maybe I could just pass in a `className` so in general I'll see about improving the nesting here.